src/index.js

```diff
@@ -172,46 +172,46 @@ $(`.main__section--hydration`).on(`click`, () => {
         flightsOfStairs: parseInt($('#form__control--stairs').val())
       })
     })
-    .then(response => response.json())
-    .then(data => console.log(data))
-    .catch(err => console.log(err));
+      .then(response => response.json())
+      .then(data => console.log(data))
+      .catch(err => console.log(err));
     $('.form-control').val('');
   } if (event.target.id === 'hydration-submit') {
-      fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData', {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json'
-        },
-        body: JSON.stringify({
-          userID: currentUserID,
-          date: $('#form__control--date2').val(),
-          numOunces: parseInt($('#form__control--water').val())
-        })
+    fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        userID: currentUserID,
+        date: $('#form__control--date2').val(),
+        numOunces: parseInt($('#form__control--water').val())
       })
+    })
       .then(response => response.json())
       .then(data => console.log(data))
       .catch(err => console.log(err));
-      $('.form-control').val('');
-    } if (event.target.id === 'sleep-submit') {
-        fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/sleep/sleepData', {
-          method: 'POST',
-          headers: {
-              'Content-Type': 'application/json'
-          },
-          body: JSON.stringify({
-            userID: currentUserID,
-            date: $('#form__control--date3').val(),
-            hoursSlept: parseInt($('#form__control--hours').val()),
-            sleepQuality: parseInt($('#form__control--quality').val())
-          })
-        })
-        .then(response => response.json())
-        .then(data => console.log(data))
-        .catch(err => console.log(err));
-        $('.form-control').val('');
-      } else {
-          return;
-        }
+    $('.form-control').val('');
```

Comment:
In theory, we shouldn't get any merge conflicts, if we do merge this in, but we may after my linting PR gets merged.